### ISSUE TITLE
rnaseq trackhub bigwig nomenclature

### ISSUE THREAD

**Issue by daler** [lcdb/lcdb-wf#179]
The trackhubs (and bigWig rules) for RNA-seq have the poor nomenclature of "sense" and "antisense". These should be changed.

The way it works now is that bigWigs are created for reads mapping to either strand in the BAM file, but this is independent of whether the assay reports the same or opposite strand reads for a transcript's origin strand. So there is the additional bigwig symlink rule that makes it easy to swap these without having to wait regenerate the bigwigs (which can take a while).

The names of the bigwigs should be chosen carefully to reflect these differences in BAM vs transcript strand.

**Comment by daler** [comment]
After some discussion, we arrived at the following:

- The existing symlink rule to rename bigwigs is not elegant, and really just serves as a way to save time on creating bigwigs when the strandedness of the kit is not known.

- There is no obvious way to name symlinked bigwigs -- pos/neg vs plus/minus vs forward/reverse are all unclear as to which should be the final one.

- No obvious way to signal to the user, via filenames, that all bigwigs should be kept. That is, adding `_final`, to have e.g., `.pos_final.bigwig`, or adding `tmp` or `intermediate` to the original bigwigs might send the wrong message that it's OK to delete. 

- Setting the original bigwigs to read-only would not be consistent with the rest of the workflow.

Fundamentally, the reason for symlinking was to handle cases where the user is not sure about kit strandedness (which, in practice, is pretty often). Re-running the bigwigs has a time and compute cost. However, performing the symlinks, documenting, and managing has a complexity and maintenance cost.

In the end, we decided to:

- remove the symlinking step
- add notes in the Snakefile clearly indicating which params to change
- update rnaseq_patterns and chipseq_patterns appropriately
- update trackhub (rnaseq and chipseq) appropriately

In addition, we proposed a downstream script that is run in order to check for possible issues. One such issue would be that the strandedness is improperly configured. The details of this script still need to be worked out, though.
